sig-edge/concepts-and-designs/kubesphere-kubeedge-integration.md

```diff
@@ -0,0 +1,80 @@
+KubeSphere & KubeEdge Integration
+
+# Summary
+
+This proposal describes KubeEdge integration with KubeSphere. The aim is to add KubeEdge support to enable edge computing feature. The bundle of integration is listed below:
+
+- Enable and configure KubeEdge CloudCore via UI
+- Edge nodes logging and metrics
+- Convenient edge node joining and exiting
+
+# E1. Enable and configure KubeEdge CloudCore via UI
+Users can toggle enable/disable and modify parameters of KubeEdge CloudClore on KubeSphere UI. Each cluster can config its KubeEdge component separately, only users with cluster-admin role can config KubeEdge. Configure parameters are shown below:
+
+## CloudCore config
+
+| Parameter | Description | Type | Default |
+| --------- | ----------- | ---- | ------- |
+| HA deploy | HA deployment of CloudCore | Bool | False |
+| Service Type | Service expose methon | String | NodePort (NodePort/LoadBalancer) |
+| CloudHub advertiseAddress | Exposed IP address for EdgeCore to connect | String | Public IP address of KS cluster in NodePort mode / Service attached LB IP |
+| CloudHub nodeLimit | node limit for edge nodes | Int | 10 |
+| CloudStream enable | Enable CloudStream component, required by logs and metrics | Bool | True |

```

Comment:
CloudStream enable => Enable CloudStream?
Param for cloudstream should be put together.
Same for CloudHub and ClouldTunnel

Reply:
Fixed